### DISCUSSION TITLE
PartiQL support in place of DQL

### DISCUSSION THREAD

**Discussion by ned-kelly** [getredash/redash#6697] [Ideas]
Given you can natively query PartiQL now via the AWS SDK - would it be much work to add a new connector called DynamoDB (PartiQL) - then users using DynamoDB can update to the newer releases that have DQL support removed?

**Comment by guidopetri** [comment]
I imagine this would be a different query runner. It shouldn't be too difficult to add / transition over, so I think we'd happily accept a PR.
